teammapper-frontend/mmp/src/map/handlers/history.ts

```diff
@@ -26,6 +27,22 @@ export default class History {
         this.snapshots = []
     }
 
+    private switchDiffKeys(snapshotDiff: MapDiff) {
```

Comment:
some comments would be nice here 😂

Reply:
Added some now, hopefully clears it up a bit :D